### ISSUE TITLE
How do I know which SKUs are being used?

### ISSUE THREAD

**Issue by ondrejromancov** [lejard-h/google_maps_webservice#38]
In my app, I would like to search for a place through a search bar with autocomplete. I would only like to search by the name of the place or address. I would then like to get the coordinates of the found place through the geocoding api. 

I would like to do all of that with minimal costs and as I do not need any detailed information about the places (opening hours, reviews, photos and etc), I would like to know how to "turn them off" or find whether they are being used.

**Comment by lejard-h** [comment]
Maybe I am wrong but I don't think the google maps API let you chose when you want the place details.

**Comment by lejard-h** [comment]
It will be fix in the next version `0.0.13`
```dart
places.getDetailsByPlaceId(placeId, fields: [ 'geometry' ]);
```

All possible fields availabel [here](https://developers.google.com/places/web-service/details#fields)

**Comment by augustinreille** [comment]
How can I use these fields in version `0.0.16`?
